tests/fixtures/saas_fixtures.py

```diff
@@ -142,27 +194,29 @@ def connection_config_saas_with_invalid_saas_config(
 
 @pytest.fixture(scope="function")
 def saas_secrets():
-    return saas_secrets_dict["mailchimp"]
+    return saas_secrets_dict
 
 
 @pytest.fixture(scope="function")
-def mailchimp_account_email():
-    return pydash.get(saas_config, "mailchimp.account_email") or os.environ.get(
-        "MAILCHIMP_ACCOUNT_EMAIL"
+def mailchimp_identity_email():
+    return pydash.get(saas_config, "mailchimp.identity_email") or os.environ.get(
+        "MAILCHIMP_IDENTITY_EMAIL"
     )
 
 
 @pytest.fixture(scope="function")
-def reset_mailchimp_data(connection_config_saas, mailchimp_account_email) -> Generator:
+def reset_mailchimp_data(
+    connection_config_mailchimp, mailchimp_identity_email
+) -> Generator:
```

Comment:
We don't necessarily have to do this here, but ultimately we're going to want to keep these fixtures in their own files akin to datastores, e.g. `../mailchimp_fixtures.py`